src/renderer/state.ts

```diff
@@ -157,6 +165,21 @@ export class AppState {
   private readonly defaultVersion: string;
   public appData: string;
 
+  // Populating versions in fiddle-core
+  // TODO(aryanshridhar): Docs recommends not to use this in production!
+  public baseVersions: BaseVersions = new BaseVersions(getElectronVersions());
```

Comment:
Calling out the TODO here: what do we not want to use in production? Is there anything we need to change here before we could ship this in Fiddle?

Reply:
This is because `BaseVersion` is [documented](https://github.com/electron/fiddle-core/blob/main/src/versions.ts#L84) to be a only used while testing.
Our use case here - If fiddle-core is not populated with versions, it tries to fetches the version from [here](https://github.com/electron/fiddle-core/blob/main/src/versions.ts#L203). This causes a couple of errors - 
* We already have this code and electron version are fetched in fiddle prior.
* Since, it uses [node-fetch](https://github.com/electron/fiddle-core/blob/main/src/versions.ts#L4), it throws an error when this function is called here. (The error that I mentioned [here](https://electronhq.slack.com/archives/C03G7RWF6KH/p1657741854218999))

Reply:
Resolving as we said it was okay in our weekly mentor meeting. 👍